webapp/channels/src/components/cloud_invoice_preview/index.tsx

```diff
@@ -70,8 +72,8 @@ function CloudInvoicePreview(props: Props) {
                             extension: 'pdf',
                             size: 0,
                             name: '',
-                        }}
-                        fileUrl={props.url}
+                        } as FileInfo}
```

Comment:
What do we need here a "fake empty file info"? shouldn't be another thing, like a null?

Reply:
should we pass `null`? I did not understand what you are conveying

Reply:
My idea is exactly that, if this is a "null" file, it shouldn't be null value, and handle it properly, instead of passing some kind of "fake null file"?